WishList not showing ratings on PC

Here is the copy/paste of devtool from Chrome (using the 'installed' desktop icon):

sync.service.ts:49 Sync: Storage will not be cleared except by explicit user action.
sync.service.ts:57 Sync: DIM is using 24.42 MB total out of 139.06 GB in storage quota (0%).
Index.tsx:68 DIM v6.9.0 (release) - Please report any errors to https://www.github.com/DestinyItemManager/DIM/issues
reducer.ts:99 Load accounts from IDB
actions.ts:118 globalSettings Object
5A cookie associated with a cross-site resource at <URL> was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at <URL> and <URL>.
actions.ts:395 [importLegacyData] Don't need to import, this legacy data has already imported
reducer.ts:53 Uncaught (in promise) TypeError: Cannot read property 'wishListAndInfo' of undefined
    at g (reducer.ts:53)
    at redux.js:459
    at reducers.ts:53
    at g (redux.js:213)
    at index.js:11
    at dispatch (redux.js:638)
    at wishlist-fetch.ts:117
    at async https:/app.destinyitemmanager.com/main-cdbcd1.js?__WB_REVISION__=ff2024cf544d41b6f8741f6ace9922c9:1
actions.ts:80 Saving profile data to IDB
manifest-service-json.ts:169 Load manifest: 9410.552001953125ms
d2-stores.ts:258 Process inventory: 1511ms
d2-stores.ts:262 Inventory state update: 3543.000244140625ms
inventory:1 [Intervention] An <img> element was lazyloaded with loading=lazy, but had no dimensions specified. Specifying dimensions improves performance. See https://crbug.com/954323
destiny-account.ts:212 Error getting D1 characters for Object Error: Bungie.net was too slow to respond.
    at p (bungie-service-helper.ts:98)
    at async m (bungie-service-helper.ts:43)
    at async d (destiny1-api.ts:23)
    at async g (destiny-account.ts:192)
    at async Promise.all (index 1)
    at async https:/app.destinyitemmanager.com/main-cdbcd1.js?__WB_REVISION__=ff2024cf544d41b6f8741f6ace9922c9:1
    at async f (destiny-account.ts:101)
    at async https:/app.destinyitemmanager.com/main-cdbcd1.js?__WB_REVISION__=ff2024cf544d41b6f8741f6ace9922c9:1

--- a/config/i18n.json
+++ b/config/i18n.json
@@ -1030,6 +1030,7 @@
     "Header": "Wish List",
     "Import": "Load Wish List Rolls",
     "ImportFailed": "No wish list information found.",
+    "ImportError": "Error loading wish list: {{error}}",
     "ImportNoFile": "No file selected.",
     "ImportSuccess": "Wish list loaded with information about one roll.{{titleAndDescription}}",
     "ImportSuccess_plural": "Wish list loaded with information about {{count, number}} rolls.{{titleAndDescription}}",

--- a/src/app/settings/WishListSettings.tsx
+++ b/src/app/settings/WishListSettings.tsx
@@ -82,7 +82,7 @@ function WishListSettings({
     reader.onload = () => {
       if (reader.result && typeof reader.result === 'string') {
         const wishListAndInfo = toWishList(reader.result);
-        dispatch(transformAndStoreWishList('', wishListAndInfo));
+        dispatch(transformAndStoreWishList(wishListAndInfo));
         ga('send', 'event', 'WishList', 'From File');
       }
     };

--- a/src/app/settings/WishListSettings.tsx
+++ b/src/app/settings/WishListSettings.tsx
@@ -16,6 +16,7 @@ import { transformAndStoreWishList, fetchWishList } from 'app/wishlists/wishlist
 import { isUri } from 'valid-url';
 import { toWishList } from 'app/wishlists/wishlist-file';
 import { settingsSelector } from './reducer';
+import { showNotification } from 'app/notifications/notifications';
 
 interface StoreProps {
   wishListsEnabled: boolean;
@@ -59,7 +60,7 @@ function WishListSettings({
     setLiveWishListSource(wishListSource);
   }, [wishListSource]);
 
-  const wishListUpdateEvent = () => {
+  const wishListUpdateEvent = async () => {
     const newWishListSource = liveWishListSource?.trim();
     if (
       newWishListSource &&
@@ -70,16 +71,23 @@ function WishListSettings({
       return;
     }
 
-    dispatch(fetchWishList(newWishListSource));
-
-    ga('send', 'event', 'WishList', 'From URL');
+    try {
+      await dispatch(fetchWishList(newWishListSource));
+      ga('send', 'event', 'WishList', 'From URL');
+    } catch (e) {
+      showNotification({
+        type: 'error',
+        title: t('WishListRoll.Header'),
+        body: t('WishListRoll.ImportError', { error: e.message })
+      });
+    }
   };
 
   const loadWishList: DropzoneOptions['onDrop'] = (acceptedFiles) => {
     dispatch(clearWishLists());
 
     const reader = new FileReader();
-    reader.onload = () => {
+    reader.onload = async () => {
       if (reader.result && typeof reader.result === 'string') {
         const wishListAndInfo = toWishList(reader.result);
         dispatch(transformAndStoreWishList(wishListAndInfo));

--- a/src/app/wishlists/actions.ts
+++ b/src/app/wishlists/actions.ts
@@ -5,7 +5,6 @@ export const loadWishLists = createAction('wishlists/LOAD')<{
   wishListAndInfo: WishListAndInfo;
   // Defaults to "now" but can be set if we're loading from IndexedDB
   lastFetched?: Date;
-  wishListSource?: string;
 }>();
 
 export const clearWishLists = createAction('wishlists/CLEAR')();

--- a/src/app/wishlists/wishlist-fetch.ts
+++ b/src/app/wishlists/wishlist-fetch.ts
@@ -52,6 +52,7 @@ export function fetchWishList(newWishlistSource?: string): ThunkResult {
     const wishListText = await wishListResponse.text();
 
     const wishListAndInfo = toWishList(wishListText);
+    wishListAndInfo.source = wishListSource;
 
     const existingWishLists = wishListsSelector(getState());
 
@@ -61,20 +62,17 @@ export function fetchWishList(newWishlistSource?: string): ThunkResult {
       existingWishLists?.wishListAndInfo?.wishListRolls?.length !==
       wishListAndInfo.wishListRolls.length
     ) {
-      dispatch(transformAndStoreWishList(wishListSource, wishListAndInfo));
+      dispatch(transformAndStoreWishList(wishListAndInfo));
     } else {
       console.log('Refreshed wishlist, but it matched the one we already have');
     }
   };
 }
 
-export function transformAndStoreWishList(
-  wishListSource: string,
-  wishListAndInfo: WishListAndInfo
-): ThunkResult {
+export function transformAndStoreWishList(wishListAndInfo: WishListAndInfo): ThunkResult {
   return async (dispatch) => {
     if (wishListAndInfo.wishListRolls.length > 0) {
-      dispatch(loadWishLists({ wishListAndInfo, wishListSource }));
+      dispatch(loadWishLists({ wishListAndInfo }));
 
       const titleAndDescription = _.compact([
         wishListAndInfo.title,
@@ -111,6 +109,13 @@ function loadWishListAndInfoFromIndexedDB(): ThunkResult {
       return;
     }
 
-    dispatch(loadWishLists(wishListState));
+    if (wishListState?.wishListAndInfo?.wishListRolls?.length) {
+      dispatch(
+        loadWishLists({
+          lastFetched: wishListState.lastFetched,
+          wishListAndInfo: wishListState.wishListAndInfo
+        })
+      );
+    }
   };
 }

--- a/src/locale/dim.json
+++ b/src/locale/dim.json
@@ -1020,6 +1020,7 @@
     "ExternalSource": "Optionally, supply the URL for a wish list",
     "Header": "Wish List",
     "Import": "Load Wish List Rolls",
+    "ImportError": "Error loading wish list: {{error}}",
     "ImportFailed": "No wish list information found.",
     "ImportNoFile": "No file selected.",
     "ImportSuccess": "Wish list loaded with information about one roll.{{titleAndDescription}}",

